common/utils/src/test/scala/org/apache/spark/util/StructuredLoggingSuite.scala

```diff
@@ -108,6 +113,18 @@ trait LoggingSuiteBase
       }
   }
 
+  test("Logging with MDC(the value is null)") {
+    Seq(
+      (Level.ERROR, () => logError(msgWithMDCValueIsNull)),
+      (Level.WARN, () => logWarning(msgWithMDCValueIsNull)),
+      (Level.INFO, () => logInfo(msgWithMDCValueIsNull))).foreach {
+      case (level, logFunc) =>
+        val logOutput = captureLogOutput(logFunc)
+        println(logOutput)
```

Comment:
let's get rid of println...

Reply:
Oh, this is my fault. I forgot to delete it in order to see what was output locally. Sorry.
Done.